### ISSUE TITLE
Wrong selector is used when promptTimeOnDateChange option is active

### ISSUE THREAD

**Issue by zippy84** [Eonasdan/tempus-dominus#2829]
### Prerequisites

- [X] I have [searched](https://github.com/Eonasdan/tempus-dominus/issues?q=is%3Aissue) for duplicate or closed issues.
- [X] I have validated my setup against the latest version on [StackBlitz](https://stackblitz.com/edit/tempus-dominus-v6-simple-setup).
- [X] I have read the [contributing guidelines](https://github.com/Eonasdan/tempus-dominus/blob/master/.github/CONTRIBUTING.md).

### Describe the issue

In TempusDominus._handleAfterChangeEvent, the selector of currentTarget is wrong.

The bug is located here:

https://github.com/Eonasdan/tempus-dominus/blob/efba21adbe7e29aa04f71e140d8f76c22c006548/src/js/tempus-dominus.ts#L503-L505

It should be this:

```js
currentTarget: this.display.widget.querySelector('[data-action="togglePicker"]'),
```

### StackBlitz fork

https://stackblitz.com/edit/tempus-dominus-v6-simple-setup-e1lshj?file=index.js

### What operating system(s) are you seeing the problem on?

Linux

### What version of are you using? You can find this information from the sample StackBlitz.

v6.7.7

### What your browser's locale? You can find this information from the sample StackBlitz.

de

**Comment by Eonasdan** [comment]
Hi. Can you see if this is fixed in 6.7.11?

**Comment by zippy84** [comment]
Looks good to me. Thanks for fixing this :+1:
